maxfw/utils/image_utils.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import division
+import sys
+from PIL import Image
+import collections
+
+from . import image_functions as F
+
+if sys.version_info < (3, 3):
+    Sequence = collections.Sequence
+    Iterable = collections.Iterable
+else:
+    Sequence = collections.abc.Sequence
+    Iterable = collections.abc.Iterable
+
+
+_pil_interpolation_to_str = {
+    Image.NEAREST: 'PIL.Image.NEAREST',
+    Image.BILINEAR: 'PIL.Image.BILINEAR',
+    Image.BICUBIC: 'PIL.Image.BICUBIC',
+    Image.LANCZOS: 'PIL.Image.LANCZOS',
+    Image.HAMMING: 'PIL.Image.HAMMING',
+    Image.BOX: 'PIL.Image.BOX',
+}
+
+
+class ImageProcessor(object):
+    """Composes several transforms together.
+
+    Args:
+        transforms (list of ``Transform`` objects): list of transforms to compose.
+
+    Example:
+        >>> pipeline = ImageProcessor([
+        >>>     Rotate(150),
+        >>>     Resize([100,100])
+        >>> ])
+        >>> pipeline.apply_transforms(img)
+    """
+
+    def __init__(self, transforms=[]):
+        assert isinstance(transforms, Iterable)
+        self.transforms = transforms
+
+    def apply_transforms(self, img):
```

Comment:
Perhaps add some doc here? What is the img? A file name, or a Pillow object, or something else?

Reply:
Thanks, updated this and will merge it in soon.